src/app/shared/components/smart-search/smart-search.component.ts

```diff
@@ -26,6 +26,7 @@ import { IOrganization } from 'iam-client-lib/dist/src/modules/domains/domains.t
 export class SmartSearchComponent implements AfterViewInit {
   @Input() searchText: FormControl = new FormControl('');
   @Input() placeholderSearch: string;
+  @Input() addOnSelect: boolean;
```

Comment:
I suppose you do not need to add this property. You can use `isAdding` getter. It's checking already if it is for adding role or not. However i see that those names do not describes what they are doing. What do you think about renaming them to be more descriptive?
I see two options:
1. Rename `add -> addRole`
2. change `add` interface, by changing `role` into `value`. Method `addRole -> addSelection` This way we can `add` organization/application by using smart search. 
What do you think?
btw. what do you think about removing `add` button from html?

Reply:
@dawidgil it looks like above it says to change add to addRole, then change addRole to addSelection. Can you clarify? I'm up for removing the add button if everyone is okay with it!

Reply:
@dawidgil Oh NM, I totally see what you mean!